R/utils_format.R

```diff
@@ -148,6 +149,12 @@
     x$Label <- NULL
   }
 
+  if (!defined_equation && inherits(attributes(x)$model, c("lavaan", "blavaan")) && "Defined" %in% x$Component) {
```

Comment:
@jmgirard I think you just need to remove `!defined_equation` here, and remove that argument in all places where you formerly added it. Then we can make this print-behaviour the default.

Reply:
Done in https://github.com/easystats/parameters/pull/472/commits/a8cde48041c0db899dfec0d350227247932040c8